src/main/java/seedu/address/logic/commands/ReviewCommand.java

```diff
@@ -21,18 +22,22 @@
 public class ReviewCommand extends Command {
 
     public static final String COMMAND_WORD = "review";
+    public static final String MESSAGE_EATERY_CLOSED = "The eatery is closed!";
+
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a review to the eatery identified "
             + "by the index number used in the last eatery listing. "
             + "New review will be added to the eatery.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_DESCRIPTION + "[DESCRIPTION]"
-            + PREFIX_COST + "[PRICE PER PAX]"
-            + PREFIX_RATING + "[RATING]\n"
+            + PREFIX_DESCRIPTION + " [DESCRIPTION] "
+            + PREFIX_COST + " [PRICE PER PAX] "
+            + PREFIX_RATING + " [RATING]"
+            + PREFIX_DATE + " [DATE]\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_DESCRIPTION + " Nice condensed milk pork ribs "
             + PREFIX_COST + " 6.20 "
-            + PREFIX_RATING + " 4";
+            + PREFIX_RATING + " 4 "
+            + PREFIX_DATE + " 14/02/2020";
```

Comment:
should specify that the format needs to be STRICTLY dd/mm/yyyy jic our users are big dumb dumbs